plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/type/TypeInfo.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.bigquery.type;
+
+public abstract sealed class TypeInfo
+        permits PrimitiveTypeInfo, ArrayTypeInfo, StructTypeInfo
+{
+    protected TypeInfo() {}
+
+    @Override
+    public abstract String toString();
```

Comment:
these methods are not needed, remove toString, equals and hashCode. toString is only used in test but that's redundant - we only need a single roundtrip: String -> TypeInfo

Reply:
`toString` is used in a few exceptions. I'll remove others.

Reply:
Where's the exceptions?

Reply:
Like here: https://github.com/nineinchnick/trino/blob/bigquery-bulk-columns/plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryTypeManager.java#L455

Reply:
Yes. I can either remove it, or we could add some logging, to make it easier to debug why some columns are not available in Trino. LMK what's better.